server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/json/Fixture.scala

```diff
@@ -23,7 +23,7 @@ import eu.timepit.refined.auto._
 import org.apache.james.jmap.model.CapabilityIdentifier.CapabilityIdentifier
 import org.apache.james.jmap.model.Id.Id
 import org.apache.james.jmap.model.Invocation.{Arguments, MethodCallId, MethodName}
-import org.apache.james.jmap.model.{ClientId, CreatedIds, Invocation, ResponseObject, ServerId}
+import org.apache.james.jmap.model._
```

Comment:
That's the only modification of your commit, I don't understand where the description `Integration test for SessionRoutes ` is?

Reply:
drop it.